<launch>
  <!-- Start the joy stick interface node -->
  <node pkg="joy" type="joy_node" name="joy_node" output="screen" 
        required="true">
    <param name="autorepeat_rate" value="20" type="double" />
    <param name="coalesce_interval" value="0.05" type="double" />
  </node>

  <!-- Start the node that converts joy messages to cmd_vel's -->
  <node pkg="segway_joy_teleop" type="segway_joy_teleop.py" name="segway_joy_teleop"
        output="screen" required="true">
    <param name="max_linear_vel" type="double" value="2.0" />
    <param name="max_angular_vel" type="double" value="2.0" />
    <remap from="~cmd_vel" to="/segwayrmp440_node/cmd_vel" />
    <remap from="~rmp_cmd" to="/segwayrmp440_node/rmp_cmd" />
  </node>
</launch>
